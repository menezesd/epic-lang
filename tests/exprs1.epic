func main() {
    print 42;
    print 2+2;
    print 3+2*2;
    print 1-10-100+1000-10000;
    print 12*2/3/4*3;
    print -5++6;
    print --5*++++1;
    print -5*-2;
    print (3+2)*2;
    print -(3+5)++(3+4-23);
    print -4%3;
    print 4%3;
    print 0/42+353%22;
    print 2*(3-4);
    print 41017513351351564217462913954*5186319567136512985612897365732896157891267965;
    print 1*2*3*4%10*2/3*97;
}