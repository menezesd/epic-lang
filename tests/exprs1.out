42
4
7
-9109
6
1
5
10
10
-24
2
1
1
-2
212729932089397786607094835434169860267629347462565051776573263560751683610
194
